src/main/java/org/ua/javaPro/berezhnoy/bank/currencyConverterAPI/CurrencyApiCurrencyConverter.java

```diff
@@ -0,0 +1,42 @@
+package org.ua.javaPro.berezhnoy.bank.currencyConverterAPI;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.Map;
+
+@Service
+public class CurrencyApiCurrencyConverter implements CurrencyConverter {
+
+    private final WebClient webClient;
+    private final PropCurrency config;
+
+    public CurrencyApiCurrencyConverter(PropCurrency config) {
+        this.webClient = WebClient.builder().build();
+        this.config = config;
+    }
+
+    @Override
+    public double convert(String from, String to, double amount) {
```

Comment:
```suggestion
    public double convert(Currency from, Currency to, double amount) {
```
це як воно звучало в ДЗ 
або ти маєш валідувати, що в строках from і to приходить дійсно валюта.